L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -21,7 +22,8 @@
                                                # Quality Flag and Quality params
                                                TrackQuality = cms.bool(True),
                                                TrackQualityPSet = cms.PSet(TrackQualityParams),
-                                               Fakefit = cms.bool(False)
+                                               Fakefit = cms.bool(False),
+                                               EmulateTB = cms.bool(False)
```

Comment:
Add comment saying that Fakefit=True causes Tracklet reco to output TTTracks before DR & KF.
Add comment saying that EmulateTB determines if TrackBuilder emulation writes EDProduct. Consider changing name of this cfg param, since current name suggests it switch on/off entirely the TrackBuilder emulation, or that it is something to to with TestBeam.

Reply:
done